src/distances.jl

```diff
@@ -104,15 +110,15 @@ function aniso2distance(semiaxes::AbstractVector, angles::AbstractVector;
   !rule.radian && (angles = deg2rad.(angles))
   _0 = zero(eltype(angles))
   N == 2 && (angles = [angles[1], _0, _0])
-  intr = @. (rule.motion == :CW) & !rule.extrinsic
+  intr = @. (rule.motion == :CW)  & !rule.extrinsic
   extr = @. (rule.motion == :CCW) & rule.extrinsic
   angles[intr .| extr] *= -1
 
+  # for reverse extrinsic transformation
+  rule.extrinsic && rev && (angles *= -1)
+
```

Comment:
So you are replacing the transposition by a flip of sign in the angle? Can you comment on this change?

Reply:
For octants, I need to reverse rotation in order the semiaxes match cartesian axes. Intrinsic rotations can be reversed only by the transpose of rotation matrix. Extrinsic case need to invert sign of the angles, that's why I included this rev kwarg and operation